includes/REST_API/Products_Controller.php

```diff
@@ -387,7 +387,7 @@ static function( $image_id ) {
 				'productPriceCurrency' => get_woocommerce_currency(),
 				'productImages'        => $images,
 				'aggregateRating'      => [
-					'ratingValue' => $product->get_average_rating(),
+					'ratingValue' => (int) $product->get_average_rating(),
```

Comment:
This looks incorrect to me.

The schema says `ratingValue` is a `number`, [which can be a floating point number](https://json-schema.org/understanding-json-schema/reference/numeric.html).

`get_average_rating()` returns a float.

This is important because the average rating could be something like 4.4. Don't want to cast this to an int and turn this into `4`.

Reply:
```suggestion
					'ratingValue' => (float) $product->get_average_rating(),
```